src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="images/logo.png"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav__menu">
+          <li class="nav__item">
+            <a
+              href="1"
+              class="nav__link is-active nav__link-1"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="1"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
    <header class="container">
      <a
        href="images/logo.png"
        class="logo"
      >
        <img
          src="images/logo.png"
          alt="moyo logo"
        />
      </a>
      
      <nav>
        <ul class="nav__menu">
          <li class="nav__item">
            <a
              href="1"
              class="nav__link is-active nav__link-1"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="1"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
```

it is a good practice to add blank line between multiline sibling elements.
fix everywhere